lib/Client.ts

```diff
@@ -479,6 +551,14 @@ export interface UpsertAccountArguments {
   members?: { email?: string; userId?: string }[];
 }
 
+interface UserToAccountArguments {
+  account: AccountIdentified;
+  users: UserIdentified[];
+}
+
+export type AddUserToAccountArguments = UserToAccountArguments;
+export type RemoveUserToAccountArguments = UserToAccountArguments;
```

Comment:
Good that you exported the types per method 🚀 

Maybe just inline the types?

```ts
export type AddUserToAccountArguments = {
  account: AccountIdentified;
  users: UserIdentified[];
};

export type RemoveUserToAccountArguments = {
  account: AccountIdentified;
  users: UserIdentified[];
}
```

A bit easier to reason about?